src/styles/fonts.css

```diff
@@ -1,11 +1,71 @@
-@each $name, $weight in (thin, ultralight, light, regular, medium, semibold, bold, extrabold, heavy),
```

Comment:
@jaruesink,

I fixed the issue with `@each` not working in this PR after upgrading the node packages. The key was to remove the `precss` package (it hasn't been updated in 4+ years), and include the postcss plugins separately.

See my `postcss.config.js` file here:

https://github.com/lambda-curry/lc-component-library/pull/69/files

https://github.com/lambda-curry/lc-component-library/pull/69

Reply:
That way, we can keep the `@each` function for things like this.

Reply:
@jaredhill4 my vote is to still remove both. if @each isn't going to be maintained either then idk if it's worth it for just these fonts